code/datums/chatmessage.dm

```diff
@@ -132,7 +152,12 @@
 			symbol = "<span style='font-size: 9px; color: #3399FF;'>*</span> "
 			size = size || "small"
 		else
-			symbol = null
+			// afd or not, we don't want to actually make people's lives harder
+			var/no_symbol_regardless = (owned_by.holder?.fakekey || (owned_by.prefs.toggles2 & PREFTOGGLE_2_ANON) || !owned_by.show_checkmark)
+			if((owned_by.IsByondMember() || owned_by.donator_level) && !no_symbol_regardless)
```

Comment:
Please do this check on `client/New()`.

iirc `IsByondMember()` does an external lookup and we dont need extra overhead on every single runechat.

Reply:
Does that mean every OOC chat also checks isbyondmember? Or maybe we already have a cache somewhere

Reply:
Yeah we cache it as `prefs.unlock_content`

Reply:
good point, I think donator level shows it regardless.

Reply:
Should be fixed